doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1728,16 +1736,62 @@ String notations
      concrete string expressed as a decimal.  They may not return
      opaque constants.
 
-The following errors apply to both string and numeral notations:
+.. note::
+   Number or string notations for parameterized inductive types can be
+   added by declaring an :ref:`abbreviation <Abbreviations>` for the
+   inductive which instantiates all parameters. See example below.
+
+The :n:`via` option applies to both string and number notations:
```

Comment:
I'd move the info on `via` to the Number Notation, where you give the syntax.  It should appear after qualid__type and before "( abstract after bignat )" (i.e., in the order it appears in the Number Notation command.  In String Notation, you can put a similar header for `via` and have it refer to the description in Number Notation.

And the sentence "... option applies" probably not needed.

Reply:
I agree, done